source/initial-dns-seedlist-discovery/tests/README.md

```diff
@@ -3,6 +3,31 @@
 This directory contains platform-independent tests that drivers can use to prove their conformance to the Initial DNS
 Seedlist Discovery spec.
 
+### Prose Tests
```

Comment:
The implication of the following prose tests are that drivers can stub their DNS results right? We should mention that here if so.

Reply:
Sounds good, just captured this.